src/socket.c

```diff
@@ -589,6 +589,23 @@ void pthreads_socket_close(zval *object, zval *return_value) {
 	threaded->store.sock->fd = PTHREADS_INVALID_SOCKET;
 }
 
+#ifdef HAVE_SHUTDOWN
+void pthreads_socket_shutdown(zval *object, zend_bool how_shutdown, zval *return_value) {
```

Comment:
the signature here is incorrect - `how_shutdown` should be `zend_long` not `zend_bool`.